.gitignore

```diff
@@ -65,3 +64,5 @@ node_modules
 .DS_Store
 venv-django
 .vscode/
+# .env
```

Comment:
Do we need this? @shakirandagire given you already added `.env`?

Reply:
@Mercy-Muchai I had put it as a title. But let me remove it since its not necessary

Reply:
@Mercy-Muchai I removed the # .env from gitignore file